website/docs/blueprints/data-analytics/spark-eks-ipv6.md

```diff
@@ -0,0 +1,153 @@
+---
+title: Spark Operator running on Amazon EKS IPv6
+sidebar_position: 6
+---
+
+This example showcases the usage of Spark Operator running on Amazon EKS in IPv6 mode. the idea is to show and demonstarte running spark workloads on EKS IPv6 cluster.
+
+## Deploy the EKS Cluster with all the add-ons and infrastructure needed to test this example
+
+The Terraform blueprint will provision the following resources required to run Spark Jobs with open source Spark Operator on Amazon EKS IPv6
+
+* A Dual Stack Amazon Virtual Private Cloud (Amazon VPC) with 3 Private Subnets and 3 Public Subnets
+* An Internet gateway for Public Subnets, NAT Gateway for Private Subnets and Egress-only Internet gateway
+* An Amazon EKS cluster in IPv6 mode (version 1.30)
+* Amazon EKS core-managed node group used to host some of the add-ons that we’ll provision on the cluster
+* Deploys Spark-k8s-operator, Apache Yunikorn, Karpenter, Prometheus and Grafana server.
+
+### Prerequisites
+
+Ensure that you have installed the following tools on your machine.
+
+1. [aws cli](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html)
+2. [kubectl](https://Kubernetes.io/docs/tasks/tools/)
+3. [terraform](https://learn.hashicorp.com/tutorials/terraform/install-cli)
+
+### Clone the repository
+
+```bash
+git clone https://github.com/awslabs/data-on-eks.git
+cd data-on-eks
+export DOEKS_HOME=$(pwd)
+```
+
+### Initialize Terraform
+
+Navigate into the example directory and run the initialization script `install.sh`.
+
+```bash
+cd ${DOEKS_HOME}/analytics/spark-eks-ipv6/terraform/
```

Comment:
```suggestion
cd ${DOEKS_HOME}/analytics/terraform/spark-eks-ipv6/
```
This path seems wrong, i get back `no such file or directory`. the suggested version worked for me.